### DISCUSSION TITLE
@slideChange等事件在小程序上获取不到参数的bug的解决方案

### DISCUSSION THREAD

**Discussion by 1783883121** [zebra-ui/zebra-swiper#25] [General]
将z-swiper.vue中的253行逻辑块暂时改为`this.$emit(event, ...args);`
如图：
![1716683298975](https://github.com/zebra-ui/zebra-swiper/assets/52774084/7b4ec8a1-35fd-4ea4-9136-2befef091c6c)
![1716683779773](https://github.com/zebra-ui/zebra-swiper/assets/52774084/0795bc55-5092-4230-8137-76353d1b72cc)

**Comment by Ru1ers** [comment]
有关于事件的使用，3.0版本已做出变更，可参考文档[事件监听机制的优化](https://swiper.zebraui.com/guide/migrate.html#_2-%E4%BA%8B%E4%BB%B6%E7%9B%91%E5%90%AC%E6%9C%BA%E5%88%B6%E7%9A%84%E4%BC%98%E5%8C%96)
